### DISCUSSION TITLE
Web Apps/PWAs saved on the home screen support?

### DISCUSSION THREAD

**Discussion by ohmygaugh-crypto** [tanujnotes/Olauncher#448] [Ideas]
I have a bunch of websites/PWAs that I saved to my android home screen through the "Add to home screen"  option in the browser. Unfortunately, Olauncher doesn't pick them up/recognize they exist.

I would love to help you make web apps supported.

Best

**Comment by tanujnotes** [comment]
https://github.com/tanujnotes/Olauncher/issues/20
